config/pepperConfig.js.ctmpl

```diff
@@ -105,4 +105,20 @@ var DDP_ENV = {
   {{end}}
 {{end}}
 
+{{if eq $study_guid "testboston"}}
+    {{if eq $environment "dev"}}
+        DDP_ENV['auth0ClientId'] = "fixme-not-set";
```

Comment:
We have a auth0 client in `dev`. Can you look in vault or in auth0 dashboard and use that? Should be the one with name `testboston-app-client`.

Reply:
@yufengwng. I don't see `testboston-app-client` in https://manage.auth0.com/dashboard/us/ddp-dev/applications

Reply:
Ah, it should be another tenant?

Reply:
Could you please invite me? The Switch tenant button is inactive for me.
![image](https://user-images.githubusercontent.com/30126907/81207896-f8503200-8fd6-11ea-9322-b30abc2462af.png)

Reply:
65401f4276d956f165d4088c2260e84a4a5fd2d8